test/built-ins/TypedArray/prototype/fill/fill-values-conversion-once.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2017 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-%typedarray%.prototype.fill
+es6id: 22.2.3.8
```

Comment:
this is not a reflection of ES6 anymore, we should get rid of this es6id tag for this test.